servers/src/proto/journal_entry_protos.proto

```diff
@@ -0,0 +1,204 @@
+package tachyon.proto;
+option java_package = "tachyon.proto";
+
+// Wraps around all types of Tachyon journal messages.
+// next available id: 25
+message JournalEntry {
```

Comment:
Is there a way to split this file up into protos for each master? (similar to how the thrift files were split up)